kiwi/schema/kiwi.rnc

```diff
@@ -24,6 +24,7 @@ namespace db  = "http://docbook.org/ns/docbook"
 namespace sch = "http://purl.oclc.org/dsdl/schematron"
 namespace nul = ""
 
+label-name = xsd:token {pattern = "[a-zA-Z]{1,8}"}
```

Comment:
8 characters is the smallest common possible label length, but is really short.
Again, this is a backwards compatibility breaking change.

Reply:
So while I do not have a better idea yet, I think this is not the best way to solve the issue.

Reply:
I agree it's not the best way. The implementation before was better but inconsistent and unstable across distributions. I couldn't come up with a better alternative so far. This includes restrictions on the label I could live with. In kiwi only rootfs_label can be customized via the XML description. From my perspective the restrictions with this PR should not hurt. I admit this is just best guess

Of course if there are better ideas how to solve the initial problem I'm open to adapt